src/providers/ec2.ts

```diff
@@ -60,11 +60,51 @@ EOF
   /opt/aws/amazon-cloudwatch-agent/bin/amazon-cloudwatch-agent-ctl -a fetch-config -m ec2 -s -c file:/tmp/log.conf || exit 2
 }
 action () {
-  if [ "$(< RUNNER_VERSION)" = "latest" ]; then RUNNER_FLAGS=""; else RUNNER_FLAGS="--disableupdate"; fi
-  sudo -Hu runner /home/runner/config.sh --unattended --url "https://{}/{}/{}" --token "{}" --ephemeral --work _work --labels "{},cdkghr:started:\`date +%s\`" $RUNNER_FLAGS --name "{}" || exit 1
+  # Define variables for injected parameters
+  taskToken="$1"
+  logGroupName="$2"
+  runnerNamePath="$3"
+  githubDomainPath="$4"
+  ownerPath="$5"
+  repoPath="$6"
+  runnerTokenPath="$7"
+  labels="$8"
+  runnerNamePath2="$9"
+  labels2="${10}"
+  runnerLevel="${11}"
+
+  # Determine the value of RUNNER_FLAGS
+  if [ "$(< RUNNER_VERSION)" = "latest" ]; then
+    RUNNER_FLAGS=""
+  else
+    RUNNER_FLAGS="--disableupdate"
+  fi
+
+  # Construct the URL and labels templates
+  url="https/$githubDomainPath/$ownerPath/$repoPath"
```

Comment:
Missing `:/`? Is `url` used somewhere? I can't see it.

Reply:
This is in draft draft state :)